### DISCUSSION TITLE
Add a toolbar element that has drop down menu with button inside

### DISCUSSION THREAD

**Discussion by dmantisk** [chrisgurney/obsidian-note-toolbar#84] [Ideas]
If you either click the element or maybe even on hover it will show a drop down menu with more buttons inside
This will allow grouping similar items together if and giving you more space in the toolbar.

Additionally this can also allow a cruder way of merging toolbars as some other discussions are talking about. In this case, when multiple toolbars override, each of them becomes a single button that expands in a drop down menu

**Comment by chrisgurney** [comment]
@dmantisk Thanks for this idea as well!

I'm planning on providing two options for nested toolbars: dropdown menus, and another for items that are simply replaced with another toolbar's items.

**Comment by chrisgurney** [comment]
@dmantisk I've added more details in my Roadmap Update: https://github.com/chrisgurney/obsidian-note-toolbar/discussions/85
